docs/transcribe.md

```diff
@@ -67,9 +66,9 @@ You can also modify an existing transcription. The technique is the same as for
 
 ## Sorting lines
 
-Line ordering is automatically calculated during [segmentation](walkthrough_segment.md).  
+Line ordering is automatically calculated during [segmentation](segment.md).  
 
-When [controlling lines order](walkthrough_segment.md)<!-- todo: add a more precise link to controlling line ordering -->, you may notice that some lines are mixed up. It is possible de manually change the order inside the "Text" panel, by clicking on the "Toggle sorting mode" button, at the top of the panel.  
+When [controlling lines order](segment.md)<!-- todo: add a more precise link to controlling line ordering -->, you may notice that some lines are mixed up. It is possible de manually change the order inside the "Text" panel, by clicking on the "Toggle sorting mode" button, at the top of the panel.  
```

Comment:
A little bit of French was wrongly included here, so changed "de" by "to"

Reply:
😆 oupsy!